config/index.ts

```diff
@@ -7,3 +7,4 @@ export const PORT = process.env.PORT || 10000
 export const SCOPES = process.env.SCOPES || 'openid'
 export const DEV_AND_STAGING_SCOPES =
   process.env.DEV_AND_STAGING_SCOPES || 'openid'
+export const SGID_RULE_NAMES = process.env.SGID_RULE_NAMES || ''
```

Comment:
What's the rationale for rule names to be defined as a separate variable as opposed to an argument to the sgID client?

Reply:
I was mirroring how `scopes` work - RP's have the options to pass in different sets of scopes for every complete sgID flow. Let's say we pass in rules as an argument to the sgID client (eg. `<some_rule_that_depends_on_pocdex.public_officer_employments>`), and if RP did not pass in `pocdex.public_officer_employments` in authUrl, the Rules API call would fail because RP has not obtained consent for the underlying referenced field.

Besides, I think having consistency in how we handle scopes and rules would be good DX.

p/s: We are using rule ID's instead of rule names to identify rules now because of Rules Engine design changes.